helm/thingsboard/values.yaml

```diff
@@ -271,6 +271,9 @@ cassandra:
   replicaCount: 1
   dbUser:
     password: setplease
+  extraEnvVars:
+  - name: JVM_OPTS
```

Comment:
Why is this needed? These affect both "normal" cassandra and the init job, right?

Reply:
It is needed because nodetool uses JMX for interaction. It affects "normal" cassandra nodes.